pkgs/applications/office/appflowy/default.nix

```
@@ -1,4 +1,4 @@
-{ stdenv, lib, fetchzip, autoPatchelfHook, makeWrapper, gtk3, openssl, copyDesktopItems, makeDesktopItem }:
+{ stdenv, lib, fetchzip, autoPatchelfHook, makeWrapper, gtk3, openssl, copyDesktopItems, makeDesktopItem, xdg-user-dirs }:
```

I suggest put each attribute in a single line here. It helps in future diffs.
```suggestion
{ lib,
  stdenv,
  fetchzip,
  autoPatchelfHook,
  makeWrapper,
  gtk3,
  openssl,
  copyDesktopItems,
  makeDesktopItem,
  xdg-user-dirs 
}:
```